python/lsst/meas/astrom/pessimistic_pattern_matcher_b_3D.py

```diff
@@ -2,11 +2,53 @@
 from __future__ import division, print_function, absolute_import
 
 import numpy as np
+from scipy.optimize import leastsq
 from scipy.spatial import cKDTree
 
 import lsst.pipe.base as pipeBase
 
 
+def _rotation_matrix_chi_sq(flattened_rot_matrix, src_pattern, ref_pattern):
+    """Compute the squared differences for least squares fitting.
+
+    Given a flattened rotation matrix, a N point pattern from the source
+    catalog and the reference pattern the sources match to, compute
+    the squared differences between the points in the two patterns after
+    rotation.
+
+    Parameters
+    ----------
+    flattened_rot_matrix : float array
+        A flattened array representing a 3x3 rotation matrix. The array is
+        flattened to comply with the API of scipy.optimize.leastsq
```

Comment:
Please document the oder of the data -- how do the 9 values in the array represent the 3x3 values in the matrix? Is it elements [0,0], [0,1], [0,2], [1,0], ... or the transpose or ...?